app/views/admin/statistics_announcement_publications/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :page_title, "Connect document" %>
+<% content_for :context, @statistics_announcement.title %>
+<% content_for :title, "Connect document" %>
+<% content_for :title_margin_bottom, 6 %>
+<% content_for :error_summary, render(Admin::ErrorSummaryComponent.new(object: @statistics_announcement)) %>
+
+<%= form_with url: admin_statistics_announcement_publication_index_path(@statistics_announcement), method: :get do |_form| %>
+  <%= render "govuk_publishing_components/components/search", {
+    name: "title",
+    value: params[:title]
+  } %>
```

Comment:
Sorry Chris i just noticed one thing 

In Niks design the search has a title 

<img width="348" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/11e1f5dd-0df4-469f-82ed-4dd88e920117">

We can add one to it like this 
```
<%= render "govuk_publishing_components/components/search", {
 name: "title",
  value: params[:title],
  label_text: "Search by title or slug"
  label_size: "xl",
} %>
```

Reply:
thanks David, good spot. I've updated to reflect this now.